robot/control/Src/modules/LEDModule.cpp

```diff
@@ -167,6 +165,10 @@ void LEDModule::setColor(uint32_t led0, uint32_t led1, uint32_t led2) {
 
     std::vector<uint8_t> data;
 
+    led0 = RGB_TO_BGR(led0);
+    led1 = RGB_TO_BGR(led1);
+    led2 = RGB_TO_BGR(led2);
```

Comment:
I would just change the shifting done on line 178-190. I'm not sure it adds much doing the flip

Reply:
Probably not. I was thinking of doing this, but now come to think of it, the improvement in readability sacrifices efficiency, 6 bitwise ANDs, 5 bitwise shifts, and 3 intermediate variables vs only 3 bitwise ANDs and 3 bitwise shifts.